Sources/TensorFlow/Core/LazyTensorTrace.swift

```diff
@@ -117,14 +159,28 @@ class LazyTensorTraceBuilder {
         return LazyTensorHandle(_lazy: result, index: 0)
     }
 
-    private func makePlaceholderTensor(
-        with handle: TFETensorHandle
-    ) -> LazyTensorHandle {
-        let cTensorHandle = handle._cTensorHandle
-        let dtype = TensorDataType(TFE_TensorHandleDataType(cTensorHandle))
-        let dtypeAttr = LazyTensorOperation.Attribute.tensorDataTypeValue(dtype)
+    /// Extract the LazyTensorOperation (if any) for this handle.
+    private static func lazyTensorOperation(_ handle: _AnyTensorHandle) -> LazyTensorOperation? {
+        guard let lazyTensorHandle = handle as? LazyTensorHandle else {
+            return nil
+        }
+        guard case let .symbolic(lazyOp, _, _)  = lazyTensorHandle.handle else {
+            return nil
+        }
+        return lazyOp
+    }
+
+    private static func makePlaceholder(with dtype: TensorDataType) -> LazyTensorOperation {
```

Comment:
* Factory methods should not form a phrases with the first argument label.
* `with` as an argument label is often regarded as not being meaningful in Swift API design, so it should be avoided.

```suggestion
    private static func makePlaceholder(dataType: TensorDataType) -> LazyTensorOperation {
```